roles/logging/tasks/filebeat.yml

```diff
@@ -52,3 +56,7 @@
 
 - name: start and enable filebeat service
   service: name=filebeat state=started enabled=yes
+  register: result
```

Comment:
I hit issues here today, adding retry made it succeed on the 3rd run. Will investigate logs and find more elegant solution today